vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+
+## Are users coming from any existing tools?
+
+![Bar graph displaying different tools used before or with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-previous-tools.png)
+
+New users were able to select all that applied. More users said they were coming from **No Previous Tool** (26%) than users who found themselves looking at Dendron after something else, with **OneNote** (25%) coming in a close second.
+
+Approximations of remaining previous (or parallel) tools:
+
+- **Notion** (23%)
+- **Google Keep** (19%)
+- **Obsidian** (18%)
+- **Evernote** (14%)
+- **Roam** (5%)
+- **Logseq** (5%)
+- **Foam** (5%)
+
+15% of respondents provided additional information under **Other**, providing a total of over **70 different tools and workflows.** Here are the top **Other** tools listed.
+
+> The percentages are in relation to the respondents in **Other**
+
+- Joplin (9%)
+- org-mode / org-roam / emacs (6%)
+- Bear (6%)
+- Apple Notes (4%)
+- Plain Markdown or text files (4%)
+- Trello / Confluence / Jira (4%)
+- Typora (3%)
+- Physical notes (2%)
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested in Dendron's publishing capabilities.
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/EQoaBI8A0ZcswKQC3UMpO/)
+- [The Open PKM Catalogue](https://pkm.dendron.so): This work-in-progress is meant to be a reference of all things PKM.
+- Improvements to [importing existing Markdown](https://wiki.dendron.so/notes/f23a6290-2dec-45dc-b616-c218ee53db6b/)
+- Better workflows, such as using [multiple vaults with private and public notes](https://wiki.dendron.so/notes/6DZiBObwhZNYRjnokQ422/)
+
+Though half of respondents said they were developers, the other half of new users came from tech and non-tech backgrounds. This includes many users where Dendron is their introduction to [VS Code](https://code.visualstudio.com/), [git, and GitHub](https://wiki.dendron.so/notes/vsAxYld4zsbD2XjKoh7OP/).
+
+Here are a few areas where we've taken action:
+
+- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/)
```

Comment:
[[provide context|dendron://private/area.growth.ref.writing-style-guide#provide-context]]
- add a sentence explaining why. example
```
- [Refreshed Getting Started Guide](https://wiki.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1/): This was a complete redesign of our getting started tutorial. We've added links to all concepts as well as step by step instructions on each option (instead of assuming prior knowledge on the user)
- ...
```